command/ca/provisionerbeta/add.go

```diff
@@ -73,7 +73,13 @@ func addCommand() cli.Command {
 **step beta ca provisioner add** <name> **--type**=ACME [**--force-cn**]
 [**--admin-cert**=<file>] [**--admin-key**=<file>] [**--admin-provisioner**=<name>]
 [**--admin-subject**=<subject>] [**--password-file**=<file>] [**--ca-url**=<uri>]
-[**--root**=<file>] [**--context**=<name>]`,
+[**--root**=<file>] [**--context**=<name>]
+
+**step beta ca provisioner add** <name> **--type**=SCEP [**--force-cn**] [**--challenge**=<challenge>] 
+[**--capabilities**=<capabilities>] [**--include-root**] [**--minimum-public-key-length**=<length>] 
```

Comment:
It think we can use `min-public-key-length`.

Reply:
Indeed; more consistent with other minimums. Fixed in 8606fdcf0560e56688309948f5c81d23f8588022.